AISKU/Tests/ApplicationInsightsDeprecatedTests.ts

```diff
@@ -150,7 +150,12 @@ export class ApplicationInsightsDeprecatedTests extends TestClass {
         if(this.successSpy.called) {
             let currentCount: number = 0;
             this.successSpy.args.forEach(call => {
-                currentCount += call[1];
+                call[0].forEach(message => {
+                    // Ignore the internal SendBrowserInfoOnUserInit message (Only occurs when running tests in a browser)
```

Comment:
As comment says, these tests where failing when running in a browser but passed fine from the command line -- this fixes that issue.